packages/ppa-nest-api/src/round-action/round-action.model.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  AutoIncrement,
+  BelongsTo,
+  Column,
+  ForeignKey,
+  PrimaryKey,
+  Table,
+} from 'sequelize-typescript';
+import { RoundModel } from 'src/round/round.model';
+import { UserModel } from 'src/user/user.model';
+import { BaseModel } from 'src/util/base.model';
+import { RoundActionEntity } from './round-action.entity';
+
+@Table
```

Comment:
wouldnt it make more sense to have the class with the DB decorators called entitiy instead of the interface and call that one model?

Reply:
I thought the same thing initially but I decided to leave it like this because the sequelize ORM calls these objects "Models" as you can see in the base model util:

```
import { Model } from 'sequelize-typescript'; // <--- they call it Model

export abstract class BaseModel<T> extends Model {
  public abstract toEntity(): T;
}
```

I think this makes more sense because it makes a clear separation between Models and Entities, where Entities are the raw representation of objects in the context of application business logic and Models are basically entities associated to a data storage mechanism (a relational database in this case).